docs/migration-guides/v6.md

```diff
@@ -0,0 +1,28 @@
+# Guide: migrating to v6
+
+If you are not on v5 yet, we recommend first following the [v5 migration guide](docs/migration-guides/v5.md).
+
+## Overview
+
+- `prefer-wait-for` was removed
+- `await-fire-event` is now called `await-async-event` with support for an `eventModule` option with `userEvent` and/or `fireEvent`
+- `await-async-event` is now enabled by default for `fireEvent` in Vue and Marko shared configs
+- `await-async-event` is now enabled by default for `userEvent` in React, Angular, and DOM shared configs
```

Comment:
This is actually enabled for all shared configs, isn't it?

Reply:
oh yes, forgot about that!